### PR TITLE
Feature/delete games

### LINKED ISSUE #49: As a game owner, I want to delete games, so that I can remove games that I don't want to lend anymore
### Definition of Done

- [ ] Add functionality to the gameRouter to delete games from the database
- [ ] Add a way for the owner of a game to remove it through the frontend

### PR DESCRIPTION
**Pull request by x183**
Added functionality to backend/frontend to remove games, fixes #49

### CODE REVIEW

File: backend/src/services/gameService.ts [hunk 1]
```diff
@@ -99,4 +99,27 @@ export const filterGames = async (filter: Filter) => {
...
+		where: {
+			id: gameID
+		},
+		select: {
+			borrow: true
+		}
+	});
+	if (!game)
+		throw new Error("Game not found");
+	const notReturnedGames = game.borrow.filter(borrow => ((!borrow.returned) && borrow.borrowStart < new Date()));
```
Comment (Krysset) [inline backend/src/services/gameService.ts:116]: I think it makes more sense to have "notReturnedGames" named "borrows" or "borrowedGames"

File: backend/src/services/gameService.ts [hunk 2]
```diff
@@ -99,4 +99,27 @@ export const filterGames = async (filter: Filter) => {
...
+	if (notReturnedGames.length > 0)
```
Comment (Krysset) [inline backend/src/services/gameService.ts:117]:
```suggestion
	const borrows = game.borrow.filter(borrow => ((!borrow.returned) && borrow.borrowStart < new Date()));
	if (borrows.length > 0)
```

Comment (Krysset) [reply backend/src/services/gameService.ts]: I think it makes more sense to have "notReturnedGames" named "borrows" or "borrowedGames"

**Review by Krysset — APPROVED** [review]
Now I'm happy :))

**Review by Oscariremma — APPROVED** [review]
LGTM👍

**Pull request merged**